src/cli/rustup_mode.rs

```diff
@@ -1515,12 +1518,6 @@ fn set_auto_self_update(cfg: &mut Cfg, auto_self_update_mode: &str) -> Result<ut
     Ok(utils::ExitCode(0))
 }
 
-#[cfg_attr(feature = "otel", tracing::instrument(skip_all))]
```

Comment:
A question for @rbtcollins: when do we decide that we need to instrument a function?